skopt/dummy_opt.py

```diff
@@ -43,16 +51,14 @@ def dummy_minimize(func, bounds, maxiter=1000, random_state=None):
         http://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.OptimizeResult.html
     """
     rng = check_random_state(random_state)
+    space = Space(dimensions)
+    X = space.rvs(n_samples=maxiter, random_state=rng)
 
-    n_params = len(bounds)
-    lb, ub = extract_bounds(bounds)
-
-    X = lb + (ub - lb) * rng.rand(maxiter, n_params)
     init_y = func(X[0])
     if not np.isscalar(init_y):
         raise ValueError(
             "The function to be optimized should return a scalar")
-    y = np.asarray([init_y] + [func(X[i]) for i in range(maxiter - 1)])
+    y = np.asarray([init_y] + [func(X[1 + i]) for i in range(maxiter - 1)])
```

Comment:
How about `[func(X[i]) for i in range(1, maxiter)]` to get rid of the +1 -1 business?